kolibri/plugins/coach/assets/src/app.js

```diff
@@ -47,7 +48,12 @@ class CoachToolsModule extends KolibriApp {
         this.store.dispatch('coachNotifications/stopPolling');
       }
       // temporary condition as we're gradually moving all promises below this line to local page handlers and therefore need to skip those that we already refactored here https://github.com/learningequality/kolibri/issues/11219
-      if (to.name && to.name === PageNames.EXAMS) {
+      if (
+        to.name &&
+        (to.name === PageNames.EXAMS ||
+          to.name === LessonsPageNames.SUMMARY ||
+          to.name === LessonsPageNames.SELECTION_ROOT)
+      ) {
```

Comment:
Apologies, @ShivangRawat30 , I should have included the whole piece to demonstrate what I mean. You need to use `.includes` with the array notation. Also, note that `,` are used instead of `||`. See an example on line 44.